content/en/news/releases/1.17.x/announcing-1.17/change-notes/index.md

```diff
@@ -120,6 +120,8 @@ These notices describe functionality that will be removed in a future release ac
 
 - **Updated** `admin log`'s `-r` flag to be shorthand for `--revision` for consistency with other commands (originally `-r` was shorthand for `--reset`). [PR #41321](https://github.com/istio/istio/pull/41321)
 
+- **Updated** `client-go` to `v1.26.1`, removing support for `azure` and `gcp` auth plugins. [PR #43101](https://github.com/istio/istio/pull/43101)
```

Comment:
You could have not backticked this and added to .spelling. Not sure if kubelogin below needs to be done, but could be done at the same time.

Reply:
From what I could tell, the convention seemed to be for versions to be in backticks, and doing that solved my spelling lint failures. I also added kubelogin to .spelling